roles/deploy/tasks/prepare.yml

```diff
@@ -26,17 +26,28 @@
     path: "{{ deploy_helper.new_release_path }}"
     state: directory
 
+- name: Copy files to new build dir
+  shell: cp -pr {{ project_source_path }}/* {{ deploy_helper.new_release_path }}
+  when: deploy_using_git_archive==false and project.repo_subtree_path is not defined
+
+- name: Remove unwanted files/folders from new release
+  shell: find {{ deploy_helper.new_release_path }} -name '{{item}}' | xargs rm -rf
+  with_items: "{{ project.project_unwanted_items | default(project_unwanted_items) }}"
+  become: true
+  become_user: "{{ web_user }}"
+  when: deploy_using_git_archive==false
```

Comment:
It would be nicer to use `block` to separate the two methods, or even put them in their own files and use `include` based on the method variable.